### PR TITLE
Feature PrivacyPolicy & Impress

### PR DESCRIPTION
**Pull request by marziman**
Added PrivacyPolicy contents and impress as initial version

### CODE REVIEW

File: pages/imprint.tsx [hunk 1]
```diff
@@ -19,7 +19,17 @@ const DataPrivacy = props => {
             <Paper className={classes.content}>
             <Typography variant="h6" display="block">Impressum</Typography>
                 <br />
-                <Typography variant="body1" display="block">TBD</Typography>
+                <Typography variant="body1" display="block"><b>Small Business Hero</b>
+                <p>Diese Seite/Webapplikation wird betrieben und herausgegeben von:</p>
+                Mehmet Arziman<br></br>
```
Comment (haugthom) [inline pages/imprint.tsx]: @cnasenberg @marziman Which person should we add here? I remember that we discussed something related to this?!

File: pages/imprint.tsx [hunk 2]
```diff
@@ -19,7 +19,17 @@ const DataPrivacy = props => {
...
+                Grünewalsdtraße 75<br></br>
```
Comment (joetjengerdes) [inline pages/imprint.tsx]: Typo in your street name

**Comment by jonathan-wiens** [comment]
@cnasenberg please also check this PR (personal info in Impressum / Datenschutzbeauftragung)

File: pages/data-privacy.tsx [hunk 3]
```diff
@@ -17,10 +17,337 @@ const DataPrivacy = props => {
...
+                    k) Einwilligung
+                    <p>Einwilligung ist jede von der betroffenen Person freiwillig für den bestimmten Fall in informierter Weise und unmissverständlich abgegebene Willensbekundung in Form einer Erklärung oder einer sonstigen eindeutigen bestätigenden Handlung, mit der die betroffene Person zu verstehen gibt, dass sie mit der Verarbeitung der sie betreffenden personenbezogenen Daten einverstanden ist.</p>
+                    
+                    <b>2. Name und Anschrift des für die Verarbeitung Verantwortlichen</b>
+                    <p>Verantwortlicher im Sinne der Datenschutz-Grundverordnung, sonstiger in den Mitgliedstaaten der Europäischen Union geltenden Datenschutzgesetze und anderer Bestimmungen mit datenschutzrechtlichem Charakter ist die:</p>
+
+                    Small Business Hero <br></br>
+                    Deutschland<br></br>
+                    E-Mail: info@smallbusinesshero.de<br></br>
+                    Website: www.smallbusinesshero.de
```
Comment (cnasenberg) [inline pages/data-privacy.tsx]: https://www.smallbusinesshero.de

**Comment by cnasenberg** [comment]
Mehmet, it's totally fine for me if you put your name there - but would you not be more than the Datenschutzbeauftragte? If would not want you to be legally responsible for everything we do here - are you sure about this? We could write Small Business Hero GbR or something - if we are a GbR anyway at the moment. But then we need to make sure that we have a written list of all maintainers of our project.

**Comment by marziman** [comment]
@cnasenberg
I will adjust the PR accordingly to remove adress and add GbR. I think also thats better.

File: pages/imprint.tsx (hunk 2 above)
Comment (marziman) [reply pages/imprint.tsx]: Corrected

**Comment by marziman** [comment]
@jonathan-wiens or @haugthom this can be merged.
Due to the importance, please prioritize it if all agree.

**Review by cnasenberg — APPROVED** [review]

File: pages/data-privacy.tsx [hunk 4]
```diff
@@ -17,10 +17,337 @@ const DataPrivacy = props => {
...
+                    
+                    k) Einwilligung
+                    <p>Einwilligung ist jede von der betroffenen Person freiwillig für den bestimmten Fall in informierter Weise und unmissverständlich abgegebene Willensbekundung in Form einer Erklärung oder einer sonstigen eindeutigen bestätigenden Handlung, mit der die betroffene Person zu verstehen gibt, dass sie mit der Verarbeitung der sie betreffenden personenbezogenen Daten einverstanden ist.</p>
+                    
+                    <b>2. Name und Anschrift des für die Verarbeitung Verantwortlichen</b>
+                    <p>Verantwortlicher im Sinne der Datenschutz-Grundverordnung, sonstiger in den Mitgliedstaaten der Europäischen Union geltenden Datenschutzgesetze und anderer Bestimmungen mit datenschutzrechtlichem Charakter ist die:</p>
+
+                    Small Business Hero <br></br>
+                    Deutschland<br></br>
+                    E-Mail: info@smallbusinesshero.de<br></br>
```
Comment (haugthom) [inline pages/data-privacy.tsx:72]: Please add a link (a tag with rel="noopener")

File: pages/data-privacy.tsx (hunk 3 above)
Comment (haugthom) [inline pages/data-privacy.tsx]: Please add a link (a tag with rel="noopener")

File: pages/data-privacy.tsx [hunk 5]
```diff
@@ -17,10 +17,337 @@ const DataPrivacy = props => {
...
+                    Website: www.smallbusinesshero.de
+                    <p></p>
+                    <b>3. Name und Anschrift des Datenschutzbeauftragten</b>
+                    <p>Der Datenschutzbeauftragte des für die Verarbeitung Verantwortlichen ist:</p>
+
+                    Mehmet Arziman<br></br>
+                    Small Business Hero<br></br>
+                    E-Mail: privacy@smallbusinesshero.de<br></br>
+                    Website: www.smallbusinesshero.de
```
Comment (haugthom) [inline pages/data-privacy.tsx]: Please add a link (a tag with rel="noopener")

File: pages/data-privacy.tsx [hunk 6]
```diff
@@ -17,10 +17,337 @@ const DataPrivacy = props => {
...
+                    Deutschland<br></br>
+                    E-Mail: info@smallbusinesshero.de<br></br>
+                    Website: www.smallbusinesshero.de
+                    <p></p>
+                    <b>3. Name und Anschrift des Datenschutzbeauftragten</b>
+                    <p>Der Datenschutzbeauftragte des für die Verarbeitung Verantwortlichen ist:</p>
+
+                    Mehmet Arziman<br></br>
+                    Small Business Hero<br></br>
+                    E-Mail: privacy@smallbusinesshero.de<br></br>
```
Comment (haugthom) [inline pages/data-privacy.tsx:80]: Do we have this email address? privacy@... ?

File: pages/data-privacy.tsx [hunk 7]
```diff
@@ -17,10 +17,337 @@ const DataPrivacy = props => {
...
+
+                    <p>Weitere Informationen und die geltenden Datenschutzbestimmungen von Google können unter https://www.google.de/intl/de/policies/privacy/ und unter http://www.google.com/analytics/terms/de.html abgerufen werden. Google Analytics wird unter diesem Link https://www.google.com/intl/de_de/analytics/ genauer erläutert.</p>
```
Comment (haugthom) [inline pages/data-privacy.tsx]: Please add a link (a tag with rel="noopener")

File: pages/data-privacy.tsx [hunk 8]
```diff
@@ -17,10 +17,337 @@ const DataPrivacy = props => {
...
+
+                    <p>LinkedIn bietet unter https://www.linkedin.com/psettings/guest-controls die Möglichkeit, E-Mail-Nachrichten, SMS-Nachrichten und zielgerichtete Anzeigen abzubestellen sowie Anzeigen-Einstellungen zu verwalten. LinkedIn nutzt ferner Partner wie Quantcast, Google Analytics, BlueKai, DoubleClick, Nielsen, Comscore, Eloqua und Lotame, die Cookies setzen können. Solche Cookies können unter https://www.linkedin.com/legal/cookie-policy abgelehnt werden. Die geltenden Datenschutzbestimmungen von LinkedIn sind unter https://www.linkedin.com/legal/privacy-policy abrufbar. Die Cookie-Richtlinie von LinkedIn ist unter https://www.linkedin.com/legal/cookie-policy abrufbar.</p>
```
Comment (haugthom) [inline pages/data-privacy.tsx]: Please add a link (a tag with rel="noopener")

File: pages/data-privacy.tsx [hunk 9]
```diff
@@ -17,10 +17,337 @@ const DataPrivacy = props => {
...
+
+                    <p>YouTube und Google erhalten über die YouTube-Komponente immer dann eine Information darüber, dass die betroffene Person unsere Internetseite besucht hat, wenn die betroffene Person zum Zeitpunkt des Aufrufs unserer Internetseite gleichzeitig bei YouTube eingeloggt ist; dies findet unabhängig davon statt, ob die betroffene Person ein YouTube-Video anklickt oder nicht. Ist eine derartige Übermittlung dieser Informationen an YouTube und Google von der betroffenen Person nicht gewollt, kann diese die Übermittlung dadurch verhindern, dass sie sich vor einem Aufruf unserer Internetseite aus ihrem YouTube-Account ausloggt.</p>
+
+                    <p>Die von YouTube veröffentlichten Datenschutzbestimmungen, die unter https://www.google.de/intl/de/policies/privacy/ abrufbar sind, geben Aufschluss über die Erhebung, Verarbeitung und Nutzung personenbezogener Daten durch YouTube und Google.</p>
```
Comment (haugthom) [inline pages/data-privacy.tsx]: Please add a link (a tag with rel="noopener")

File: pages/data-privacy.tsx [hunk 10]
```diff
@@ -17,10 +17,337 @@ const DataPrivacy = props => {
...
+
+                    <p>PayPal gibt die personenbezogenen Daten gegebenenfalls an verbundene Unternehmen und Leistungserbringer oder Subunternehmer weiter, soweit dies zur Erfüllung der vertraglichen Verpflichtungen erforderlich ist oder die Daten im Auftrag verarbeitet werden sollen.</p>
+
+                    <p>Die betroffene Person hat die Möglichkeit, die Einwilligung zum Umgang mit personenbezogenen Daten jederzeit gegenüber PayPal zu widerrufen. Ein Widerruf wirkt sich nicht auf personenbezogene Daten aus, die zwingend zur (vertragsgemäßen) Zahlungsabwicklung verarbeitet, genutzt oder übermittelt werden müssen. </p>
+
+                    <p>Die geltenden Datenschutzbestimmungen von PayPal können unter https://www.paypal.com/de/webapps/mpp/ua/privacy-full abgerufen werden.</p>
```
Comment (haugthom) [inline pages/data-privacy.tsx]: Please add a link (a tag with rel="noopener")

File: pages/data-privacy.tsx [hunk 11]
```diff
@@ -17,10 +17,337 @@ const DataPrivacy = props => {
...
+
+                    <p>Betreibergesellschaft von YouTube ist die Google Ireland Limited, Gordon House, Barrow Street, Dublin, D04 E5W5, Ireland.</p>
+
+                    <p>Durch jeden Aufruf einer der Einzelseiten dieser Internetseite, die durch den für die Verarbeitung Verantwortlichen betrieben wird und auf welcher eine YouTube-Komponente (YouTube-Video) integriert wurde, wird der Internetbrowser auf dem informationstechnologischen System der betroffenen Person automatisch durch die jeweilige YouTube-Komponente veranlasst, eine Darstellung der entsprechenden YouTube-Komponente von YouTube herunterzuladen. Weitere Informationen zu YouTube können unter https://www.youtube.com/yt/about/de/ abgerufen werden. Im Rahmen dieses technischen Verfahrens erhalten YouTube und Google Kenntnis darüber, welche konkrete Unterseite unserer Internetseite durch die betroffene Person besucht wird.</p>
```
Comment (haugthom) [inline pages/data-privacy.tsx]: Please add a link (a tag with rel="noopener")

File: pages/imprint.tsx [hunk 12]
```diff
@@ -19,7 +19,17 @@ const DataPrivacy = props => {
...
-                <Typography variant="body1" display="block">TBD</Typography>
+                <Typography variant="body1" display="block"><b>Small Business Hero</b>
+                <p>Diese Seite/Webapplikation wird betrieben und herausgegeben von:</p>
+                Small Business Hero GbR<br></br>
+                Grünewaldstraße 75<br></br>
+                70469 Stuttgart<br></br>
+                +(49)0711 184 29 271<br></br>
+
+                <p>Kontakt:<br></br>
+                info@smallbusinesshero.de
```
Comment (haugthom) [inline pages/imprint.tsx:30]: Please add a link (a tag with rel="noopener")

File: pages/imprint.tsx [hunk 13]
```diff
@@ -19,7 +19,17 @@ const DataPrivacy = props => {
             <Paper className={classes.content}>
             <Typography variant="h6" display="block">Impressum</Typography>
                 <br />
-                <Typography variant="body1" display="block">TBD</Typography>
+                <Typography variant="body1" display="block"><b>Small Business Hero</b>
+                <p>Diese Seite/Webapplikation wird betrieben und herausgegeben von:</p>
+                Small Business Hero GbR<br></br>
+                Grünewaldstraße 75<br></br>
+                70469 Stuttgart<br></br>
+                +(49)0711 184 29 271<br></br>
```
Comment (haugthom) [inline pages/imprint.tsx:27]: Please add a link (a tag with rel="noopener" and "tel:" in front of value)

**Review by haugthom — APPROVED** [review]

**Pull request merged**